contributors.json

```diff
@@ -1,20 +1,85 @@
 [
   {
     "avatarUrl": "https://avatars.githubusercontent.com/u/42497995?v=4",
-    "name": "Androz",
-    "description": "God",
-    "bugs": 10,
+    "name": "@androz2091",
+    "description": "Back-end API developer",
+    "bugs": 1000000,
     "translation": "fr",
-    "developer": 10,
+    "developer": 100000,
     "priority": 0
   },
   {
     "avatarUrl": "https://avatars.githubusercontent.com/u/69633530?v=4",
-    "name": "Florian Lefebvre",
-    "description": "God after God",
+    "name": "@florian_lefebvre",
+    "description": "Front-end UI developer",
     "bugs": 10,
-    "translation": "gb-ukm",
+    "translation": "fr",
     "developer": 10,
     "priority": 1
+  },
+  {
+    "avatarUrl": "https://cdn.discordapp.com/avatars/236952680201715714/9f8f56730a097338968b19016b43e58c.webp?size=40",
```

Comment:
hardcoding discord avatar url might not be a good idea 🤔

Reply:
github avatars can be github.com/username.png which will always exist but this is not the case for discord avatars

Reply:
Yes @skdhg indeed. We'll host a low quality avatar in this repository